db/migrate/20150508124600_copy_unmanaged_hosts_to_interfaces.rb

```diff
@@ -19,9 +19,13 @@ def up
     say "Migrating Unmanaged Host interfaces to standalone Interfaces"
 
     FakeHost.where(:managed => false).each do |host|
+      say "  Migrating #{host.name}"
       nic = FakeNic.where(:host_id => host.id, :primary => true, :provision => true).first
       if nic.present?
         nic.name = host.name if nic.name.blank?
+        # we ignore validation errors on existing primary interface
+        say "Host #{host.name} has invalid interface #{nic.identifier}, you have to fix it manually through API" unless nic.valid?
```

Comment:
Yeah, at least it would be useful to `say` the nic.errors so we know precisely what's invalid.  We also know from elsewhere that `nic.identifer` often isn't populated, so a bit more info may be needed to identify it.

I think the other PR adds proper editing in the UI for these, so we won't have to rely on the API, right?